articles/frontdoor/front-door-http2.md

```diff
@@ -19,6 +19,9 @@ Currently, HTTP/2 support is active for all Front Door configurations. No furthe
 
 HTTP/2 is a major revision to HTTP/1.1. It provides faster web performance, reduced response time, and improved user experience, while maintaining the familiar HTTP methods, status codes, and semantics. Though HTTP/2 is designed to work with HTTP and HTTPS, many client web browsers only support HTTP/2 over Transport Layer Security (TLS).
 
+>[!NOTE]
+>HTTP/2 protocol support is available to clients connecting to Azure Front Door Service only. The communication to backends in the backend pool is over HTTP/1.1. 
```

Comment:
There was change in branding sometime back and we no more use Service in the name. Please use "Azure Front Door" instead of "Azure Front Door Service"

Reply:
Also, grammatically the statement is incorrect. Please change to below:
_HTTP/2 protocol support is only available for requests from clients to Front Door. The communication from Front Door to backends in the backend pool happens over HTTP/1.1._